npm/react/cypress/plugins/index.js

```diff
@@ -17,7 +19,7 @@ const webpackConfig = {
       {
         test: /\.(js|jsx|mjs|ts|tsx)$/,
         loader: 'babel-loader',
-        options: babelConfig,
+        options: { ...babelConfig, cacheDirectory: path.resolve(__dirname, '..', '..', '.babel-cache') },
```

Comment:
What is this? Is this also to speed perf up?

Reply:
And show users how to optimize theirs.